.gitignore

```diff
@@ -29,4 +29,6 @@ local-chart/
 # word counts
 /counts
 
-
+# Hub Chart generated dependencies
+hub-chart/Chart.lock
+hub-chart/charts/
```

Comment:
This PR currently includes core-chart/charts/kubeflex-operator-v0.6.1.tgz . That probably should also be absent from the PR and listed in .gitignore.

Reply:
thank you for catching it